src/components/item_bids/ItemBids.js

```diff
@@ -14,16 +14,17 @@ const ItemBids = ({ item_uuid, apiHandler, handleError, fetchBids }) => {
 
   return bids.length > 0 && (
     <>
-    <h2 className='header--secondary'>Bids</h2>
-    {bids.map(bid => {
-      return <BidCard 
-        white={true} 
-        key={bid.uuid} 
-        timestamp={bid.timestamp} 
-        bid_price={bid.bid_price}
-        userName={"Anonymous"} />
-    })}
-  </>
+      <h2 className='header--secondary'>Bids</h2>
+      {bids.map(bid => {
+        return <BidCard
+          white={true}
+          key={bid.uuid}
+          timestamp={bid.timestamp}
+          bid_price={bid.bid_price}
+          userName={"Anonymous"}
```

Comment:
Why is this hardcoded to be `"Anonymous"`? Are you waiting for some backend work to be completed?

Reply:
We wanted to anonymize the item bids, so we have it hard coded to "Anonymous" for Item Bids. This Bid Card is used for other cases as well, where we inject different messages